clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/compute/RegionalOperation.java

```diff
@@ -0,0 +1,36 @@
+package com.netflix.spinnaker.clouddriver.google.compute;
+
+import com.google.api.services.compute.model.Operation;
+import com.netflix.spinnaker.clouddriver.data.task.Task;
+import com.netflix.spinnaker.clouddriver.google.deploy.GCEUtil;
+import com.netflix.spinnaker.clouddriver.google.deploy.GoogleOperationPoller;
+import com.netflix.spinnaker.clouddriver.google.security.GoogleNamedAccountCredentials;
+
+class RegionalOperation implements WaitableComputeOperation {
+
+  private final Operation operation;
+  private final GoogleNamedAccountCredentials credentials;
+  private final GoogleOperationPoller poller;
+
+  RegionalOperation(
+      Operation operation,
+      GoogleNamedAccountCredentials credentials,
+      GoogleOperationPoller poller) {
+    this.operation = operation;
+    this.credentials = credentials;
+    this.poller = poller;
+  }
+
+  @Override
+  public Operation waitForDone(Task task, String phase) {
+    return poller.waitForRegionalOperation(
+        credentials.getCompute(),
+        credentials.getProject(),
+        operation.getRegion(),
+        operation.getName(),
+        /* timeoutSeconds= */ null,
```

Comment:
What's the rationale for keeping this null vs making it a parameter?

Reply:
Just YAGNI... if I ever get to a point where we need it, I'll make it a parameter then. I didn't check all the call sites, but I checked more than a handful and I never saw any that actually used a value other than null.